packages/create-cloudflare/templates/openapi/hono/package.json

```diff
@@ -3,15 +3,18 @@
 	"version": "0.0.1",
 	"private": true,
 	"scripts": {
+		"cf-typegen": "wrangler types",
 		"deploy": "wrangler deploy",
 		"dev": "wrangler dev",
-		"start": "wrangler dev",
-		"cf-typegen": "wrangler types"
+		"start": "wrangler dev"
 	},
 	"dependencies": {
-		"@cloudflare/itty-router-openapi": "^1.0.1"
```

Comment:
Is this really needed?

Reply:
yes, this old dependency needs to be removed